cmd/critest/util.go

```diff
@@ -29,23 +29,31 @@ import (
 
 var buildTargets = []string{
 	"github.com/kubernetes-incubator/cri-tools/pkg/validate",
+	"github.com/kubernetes-incubator/cri-tools/pkg/benchmark",
 	"github.com/kubernetes-incubator/cri-tools/vendor/github.com/onsi/ginkgo/ginkgo",
 }
 
 // build cri-tools binaries.
-func build() error {
+func build(benchmark bool) error {
 	glog.Infof("Building cri-tools binaries...")
 	outputDir, err := getBuildOutputDir()
 	if err != nil {
 		glog.Fatalf("Failed to get build output directory: %v", err)
 	}
 
-	err = runCommand("go", "test", "-c", "-v", "-o", filepath.Join(outputDir, "e2e.test"), buildTargets[0])
-	if err != nil {
-		return fmt.Errorf("failed to build e2e.test %v", err)
+	if benchmark {
```

Comment:
`runCommand` is not only for building test, but also used to run other commands like building and running ginkgo. So, is it fine to add a parameter only for building test to a common function?